app/components/site-chrome/template.hbs

```diff
@@ -109,23 +109,23 @@
           {{/if}}
         </li>
 
-        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav" }}
+        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav"}}
           {{link-to 'All Streams' 'stream' class="inherit" preventDefault=false}}
         {{/link-to}}
 
-        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href="false" class="list-item list-item--nav" }}
+        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href=false class="list-item list-item--nav"}}
```

Comment:
Minor quibble slash question for my own future coding: do we have a rule in our codebase for when to use singe vs. double quotes? It looks inconsistent in this file, for example `current-when` on this line vs. L120.